<?xml version="1.0" encoding="UTF-8"?>

<!-- #### DESCRIPTION #### -->
<!-- Run laser_scan_matcher package  -->

<launch>

    <arg name="scan_topic" value="/my_robot/front_laser_scan"/>
    <arg name="base_frame" value="base_footprint"/>
    <arg name="odom_frame" value="odom"/>

    <node pkg="laser_scan_matcher" type="laser_scan_matcher_node" name="laser_matcher" output="screen">
        <remap from="scan" to="$(arg scan_topic)"/> 
        <param name="base_frame" value="$(arg base_frame)"/>
        <param name="fixed_frame" value="$(arg odom_frame)"/>
        <param name="base_frame_id" value="$(arg base_frame)"/> 
        <param name="use_imu" value="false"/>
        <param name="use_odom" value="false"/>
        <param name="use_vel" value="false"/>
        <param name="max_iterations" value="10"/>
        <param name="publish_tf" value="true"/>
        <param name="publish_pose" value="false"/>
        <param name="publish_pose_stamped" value="true"/>
    </node>

</launch>


